lib/node/nodes/line-sequential.js

```diff
@@ -43,3 +43,63 @@ var routingSequentialQueryTemplate = Node.template([
     '  {{? it.category_column }}GROUP BY {{=it.category_column}}{{?}}',
     ') _cdb_analysis_line_sequential'
 ].join('\n'));
+
+var countDistinctValues = Node.template([
+    'SELECT COUNT(*) AS count_distict_values FROM (',
+    '  SELECT DISTINCT {{=it.category_column}}',
+    '  FROM (',
+    '    {{=it.source}}',
+    '  ) AS _line_sequential',
+    ') AS _cdb_analysis_count_distinct'
+].join('\n'));
+
+LineSequential.prototype.computeRequirements = function(databaseService, limits, callback) {
+    var inputRows = this.source.estimatedRequirements.numberOfRows;
+    var numCategories = 1;
+    var self = this;
+    var rowsLimit = Node.getNodeLimit(limits, TYPE, 'maximumNumberOfRows', 1000000);
+    var pointsLimit = Node.getNodeLimit(limits, TYPE, 'maximumNumberOfPointsPerLine', 10000);
+    function storeRequirements(err) {
```

Comment:
For the same reasons mention above, I'd prefer to keep this as is (I consider this throw-away code have our limits working until we implement proper limit-checking)